CMakeLists.txt

```diff
@@ -180,7 +189,9 @@ set(CMAKE_AUTOMOC TRUE)
 # Find other needed libraries
 FIND_PACKAGE(Protobuf REQUIRED)
 IF(NOT EXISTS "${Protobuf_PROTOC_EXECUTABLE}")
-  MESSAGE(FATAL_ERROR "No protoc command found!")
+    MESSAGE(FATAL_ERROR "No protoc command found!")
+ELSE()
+    MESSAGE(STATUS "Protoc found!")
```

Comment:
Can you specify what version you found?

Reply:
Whoops, thought I had already pushed this, should be updated now.